### PR TITLE
Editable Reblogs: restore save button dropdown options

### LINKED ISSUE #734: No option to schedule posts with Editable Reblogs
As of Editable Reblogs 3.0.0, the default dropdown buttons have been replaced with three buttons for Draft, Queue, and Post. The options for posting privately and scheduling posts are missing.

### PR DESCRIPTION
**Pull request by bvtsang**
- [x] Restore dropdown options
- [x] Restore functionality to publish, queue, and draft
- [x] Restore functionality to saving privately
- [x] Restore functionality to scheduling posts
- [x] Test that this works on the dashboard
- [x] Test that this works in drafts
- [x] Test that this works in the queue
- [x] Test that this works from `/blog/username`
  - Should show "save" instead of "post"
- [x] Test that this works from the search page
  - It doesn't
  - Shouldn't block this from going out
  - ~~Need to log issue for this~~ #788 
- [x] Test that this works from the peepr
  - Works for people you follow
  - Doesn't work for people you don't follow
  - Shouldn't block this from going out
  - ~~Need to log issue for this~~ #789
- [x] Squash commits
- [x] Investigate various ER bug reports
  - [x] Reblogging music posts blanks out posts
    - [ER enabled](http://heymrargent.tumblr.com/post/133094171057/kayayeteae-oh-my-feels-just-like-i-dont-try)
    - [ER disabled](http://heymrargent.tumblr.com/post/133094196857/kayayeteae-oh-my-feels-just-like-i-dont-try)
    - Unable to reproduce
  - [x] From the [support Gitter](https://gitter.im/new-xkit/support?at=5646a1975b7a2ea84f082079): "I can't reblog photos. It won't show on the dashboard or my account when I have editable reblogs on."
    - Unable to reproduce
  - [x] #767 
    - Is legitimate, but can be excluded from this PR
- [x] Print the actual XER-SR error message into the console

Since this pull request is only for restoring the dropdown buttons, it may not be in our best interest to include all Editable Reblogs-related fixes as part of this PR.

Fixes #734

### CODE REVIEW

**Comment by blackjackkent** [comment]
Taking a leaf from your book and adapting your test case list:

### Test Cases

For the Rich Text Editor and HTML editors, for text, photos, links, audio, and video (for posting, queue, drafting, scheduling, posting privately):

- Text
  - [x] Preserves text beneath inserted images
  - [x] Preserves post titles
  - [x] Can edit text
  - [x] Can edit text and add formatting
  - ~~Can edit titles~~ (not currently available)
- [x] \(Nested\) blockquotes
- Inserted images
  - [x] Via rich text editor button
  - [x] Via dragging and dropping an image into the editor
- [x] Video
- [x] Horizontal lines
- Read mores
  - [x] Read mores don't convert to horizontal lines
  - [x] ...for multiple read mores in the same thread
- Miscellaneous
  - [ ] Preserve original blockquoted captions (e.g. don't substitute username of original poster with the reblogger's)

For the Markdown editor:

- [x] A warning should come up telling them to switch editors

When creating a new post, Editable Reblogs shouldn't run

- [x] Click on creating a new post from the middle of the screen
- [x] Click on creating a new post from the top-right corner
- [x] Click on a notification in the activity page with One-Click Reply enabled

**Comment by nightpool** [comment]
also test that this works on already posted posts from the /blog/whatever page. (should show "save" instead of "post")

File: Extensions/editable_reblogs.js [hunk 1]
```diff
@@ -192,9 +300,9 @@ XKit.extensions.editable_reblogs = new Object({
 		request["is_rich_text[one]"] = "0";
 		request["is_rich_text[two]"] = "1";
 		request["is_rich_text[three]"] = "0";
-		request["post[slug]"] = "";
+		request["post[slug]"] = XKit.extensions.editable_reblogs.post_slug_metadata;
 		request["post[draft_status]"] = "";
```
Comment (nightpool) [inline Extensions/editable_reblogs.js]: post[draft_status] should be preserved?

Comment (nightpool) [inline Extensions/editable_reblogs.js]: looks like this doesn't do anything—editing draft posts works fine.

**Comment by nightpool** [comment]
breaks videos:
image

breaks hrs:
image
(vs)
image

**Comment by bvtsang** [comment]
To track the two bugs, I've filed #780 and #781; these bugs shouldn't block this PR from going out.

**Comment by blackjackkent** [comment]
This is ready to go for the purpose it serves. Great work @bvtsang :)

@homu r+

**Pull request merged**